src/common/constants/profile-screen.ts

```diff
@@ -0,0 +1,6 @@
+import { Dimensions } from 'react-native';
+
+const PROFILE_NAVBAR_WIDTH = Dimensions.get('screen').width * 0.8;
+const BUTTON_PROFILE_NAVBAR_WIDTH = Math.ceil(PROFILE_NAVBAR_WIDTH / 3);
```

Comment:
Rename this file to app.ts and leave here only screen width
```suggestion
const SCREEN_WIDTH = Dimensions.get('screen').width;
```

Let's put button width constant in navbar/constants.ts file:
```
import { SCREEN_WIDTH } from 'src/common/constants';
import { DEFAULT_SCREEN_PADDING } from 'src/styles';

const NAVBAR_BUTTON_WIDTH = Math.ceil(
  (SCREEN_WIDTH - DEFAULT_SCREEN_PADDING * 2) / 3
);

export { NAVBAR_BUTTON_WIDTH };
```